src/community/advanced-data-card.css.ts

```diff
@@ -84,7 +84,7 @@ export const touchableCardOverlay = style({
                 [`${touchableContainer}:hover &`]: {
                     backgroundColor: skinVars.colors.backgroundContainerHover,
                 },
-                [`${touchable}:active &`]: {
+                [`${touchableContainer}:active &`]: {
```

Comment:
This breaks the active state when card is interactive and you have top actions. When you click on the top action button, the entire card reacts to the click and that is not correct